textdb/textdb-gui/js/onstart.js

```diff
@@ -36,6 +36,18 @@ var setup = function(){
 	/*
 		Helper Functions
 	*/
+	//Process Query Helper Function
+	function createBlackBox(message){
+		var blackBox = $('<div class="black-box"><div class="result-box"><div class="result-box-band">Return Result<div class="black-box-close"><img src="img/close-icon.png"></div></div><div class="return-result"></div></div></div>');
```

Comment:
If this is a hard coded constant, the name of the variable should be all capitals. And leave a small description of the HTML constant.